testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -168,6 +168,15 @@ public void testRangePredicate()
                 "WHERE orderkey BETWEEN 10 AND 50");
     }
 
+    @Test
+    public void testPredicateReflectedInExplain()
+    {
+        // Even if the predicate is pushed down into the table scan, it should still be reflected in EXPLAIN (via ConnectorTableHandle.toString)
+        assertExplain(
+                "EXPLAIN SELECT name FROM nation WHERE nationkey = 42",
+                "(predicate|filterPredicate|constraint).{0,10}nationkey");
```

Comment:
What is the actual constraint string?

Reply:
it's `constraint on [nationkey]`

since it's a generic test, i cannot hard-code an expected value

and it's generic, since i want to enforce the behavior for all connectors.